tests/components/zha/zha_devices_list.py

```diff
@@ -1832,13 +1827,21 @@
                 "profile_id": 260,
             }
         },
-        "entities": ["light.lumi_lumi_router_77665544_on_off_on_off"],
+        "entities": [
+            "binary_sensor.lumi_lumi_router_77665544_on_off",
+            "light.lumi_lumi_router_77665544_on_off",
+        ],
         "entity_map": {
+            ("binary_sensor", "00:11:22:33:44:55:66:77-8-6"): {
+                "channels": ["on_off", "on_off"],
+                "entity_class": "Opening",
+                "entity_id": "binary_sensor.lumi_lumi_router_77665544_on_off",
+            },
             ("light", "00:11:22:33:44:55:66:77-8"): {
                 "channels": ["on_off", "on_off"],
                 "entity_class": "Light",
-                "entity_id": "light.lumi_lumi_router_77665544_on_off_on_off",
-            }
+                "entity_id": "light.lumi_lumi_router_77665544_on_off",
```

Comment:
hrm... is this a breaking change ultimately? Are entity ids changing as a result of the refactor? If so we should note it in the PR.

Reply:
Nope, it's not a breaking change, because we're keeping the `unique_id` the same, so all existing entities are going to retain their entity_id. Only newly joined devices are going to have entity_id different.

Reply:
👍